src/app/contextBroker/contextBroker.cpp

```diff
@@ -862,6 +862,7 @@ int main(int argC, char* argV[])
   paConfig("log file line format",          (void*) LOG_FILE_LINE_FORMAT);
   paConfig("log file time format",          (void*) "%Y-%m-%dT%H:%M:%S");
   paConfig("builtin prefix",                (void*) "ORION_");
+  paConfig("prefix",                        (void*) "ORION_");
```

Comment:
This here is the fix - the env vars existed and were working, they just didn't have the prefix `ORION_`